package.json

```diff
@@ -84,13 +84,13 @@
 				},
 				{
 					"command": "stitch.showTree",
-					"when": "stitch:contextAvailable || resourceFilename =~ /.integration.json$/"
+					"when": "stitch:contextAvailable || resourceFilename =~ /.integration.json$/ || resourceFilename =~ /.integration.yaml$/"
```

Comment:
Don't we need to support .yml as well?

Might want to use a regex like .(json|yml|yaml)$ then :)